sdk/samples/iot/paho_iot_hub_sas_telemetry_sample.c

```diff
@@ -151,7 +151,7 @@ void connect_mqtt_client_to_iot_hub()
   if (*az_span_ptr(env_vars.x509_trust_pem_file_path)
```

Comment:
Is it feasible for this to use `az_span_size(...) != 0`? If so, that is the preferred check.

Otherwise, nit, use the `NULL` keyword, rather than comparing to the null byte.

Reply:
making change in new PR